optics-core/src/Optics/At/Core.hs

```diff
@@ -366,10 +368,45 @@ class Ixed m => At m where
   -- >>> at 1 ?~ "hello" $ Map.empty
   -- fromList [(1,"hello")]
   --
+  -- /Note:/ Usage of this function might introduce space leaks if you're not
+  -- careful to make sure that values put inside the 'Just' constructor are
+  -- evaluated. If this is the case, use 'at'' instead.
+  --
   -- /Note:/ 'Map'-like containers form a reasonable instance, but not
   -- 'Array'-like ones, where you cannot satisfy the 'Lens' laws.
   at :: Index m -> Lens' m (Maybe (IxValue m))
 
+-- | Version of 'at' strict in the value inside the `Just` constructor.
+--
+-- Example:
+--
+-- >>> (at () .~ Just (errorWithoutStackTrace "oops") $ Nothing) `seq` ()
+-- ()
+--
+-- >>> (at' () .~ Just (errorWithoutStackTrace "oops") $ Nothing) `seq` ()
+-- *** Exception: oops
```

Comment:
Is there any reason this couldn't just be `error`, and use `...` to satisfy doctest?
```
-- >>> (at' () .~ Just (error "oops") $ Nothing) `seq` ()
-- *** Exception: oops
-- ...
```

Reply:
Oh, I didn't know that trick. Yes, doing that makes sense.